package proto_ff;

// 这里一定要包含自定义的协议
import "yd_fieldoptions.proto";
import "proto_common.proto";

enum enConstType
{
  EN_CONST_NONE													= 0;
  EN_CONST_USER_INIT_JETTON 						= 1	 [(yd_fieldoptions.field_enum_cname) = "玩家初始赠送金币"];
  EN_CONST_USER_INIT_FACE 							= 2	 [(yd_fieldoptions.field_enum_cname) = "玩家初始头像"];
  EN_CONST_USER_INIT_BANK_JETTON 				= 3	 [(yd_fieldoptions.field_enum_cname) = "玩家初始银行密码"];
  EN_CONST_USER_ACH_WITH_FEE		 				= 4	 [(yd_fieldoptions.field_enum_cname) = "=0 抽水算个流水， =1 抽水不算流水"];
  EN_CONST_USER_FEE_WITH_POUR 				  = 5	 [(yd_fieldoptions.field_enum_cname) = "=0时为不加本金的抽水， =1时为加上本金的抽水"];
  EN_CONST_USER_MIN_BANK_DRAW 				  = 6	 [(yd_fieldoptions.field_enum_cname) = "非VIP转VIP最小转账额度"];
}

message ConstDesc
{
  optional int32 id = 1[(yd_fieldoptions.field_cname) = "编号",(yd_fieldoptions.db_field_comment) = "编号", (yd_fieldoptions.db_field_type) = E_FIELDTYPE_PRIMARYKEY, (yd_fieldoptions.db_field_auto_increment)=true, (yd_fieldoptions.db_field_auto_increment_value)=1];
  optional string value = 2[(yd_fieldoptions.field_cname) = "常量值",(yd_fieldoptions.db_field_comment) = "常量值", (yd_fieldoptions.field_bufsize)=128, (yd_fieldoptions.db_field_bufsize)=128];
  optional string type_name = 3[(yd_fieldoptions.field_cname) = "类型名称",(yd_fieldoptions.db_field_comment) = "类型名称", (yd_fieldoptions.no_field)=true, (yd_fieldoptions.db_field_bufsize)=128];
}

message Sheet_ConstDesc
{
  repeated ConstDesc ConstDesc_List = 1 [(yd_fieldoptions.field_arysize)=100];
}